src/main/java/core/basesyntax/Diamond.java

```diff
@@ -6,13 +6,36 @@
 
     /**
      * Нарисовать ромб со звёздочек заданной величины.
-     *
      * Пример: величина 3, значит максимальное кол-во звёзд в ряду (по диагонали) будет 3.
-     *
      * Вместо System.out.print() или System.out.println() использовать stream.print()
      * или stream.println()
      */
-    public static void drawDiamond(int maxSize, PrintStream stream) {
+    public static void drawDiamond(int n, PrintStream stream) {
```

Comment:
try to simplify this method and try to use more understandable name of the variables